src/style.css

```diff
@@ -1,3 +1,83 @@
+
 body {
   margin: 0;
+padding: 0;
+background-color: #fff;
+border: 0;
+font-family: Roboto, sans-serif;
 }
+
+header {
```

Comment:
you should add styling to class selectors instead of tag ones

Reply:
Still not fixed